src/contributing/void-docs/style-guide.md

```diff
@@ -99,6 +99,22 @@ They should not be formatted like this:
 [https://www.example.com/](https://www.example.com/)
 ```
 
+### Checking links
+
+If you're including new links (either internal or external) in the docs or
+changing the current file structure, it is recommended to install
+`mdbook-linkcheck`, which can be obtained from the Void repos or using `cargo`.
+You can then build the mdBook locally, which will run a linkcheck as well, or
+run it in standalone mode:
+
+```
+$ mdbook-linkcheck -s
+```
+
+This way, linkcheck will verify all the references, and warn you if there are
+any issues. If any link you're using is correct but generating errors for some
+reason, you can add it to the exclude list in `book.toml`.
+
```

Comment:
Any reason(s) we shouldn't make this non-optional, like `vmdfmt`, rather than just 'recommended'? If not, how about something like:

"If adding links or changing the link structure, use \`mdbook-linkcheck\`. \`mdbook-linkcheck\` verifies all links, both internal and external. Install the \`mdbook-linkcheck\` package, then do:

\```
$ mdbook-linkcheck -s
\```

If there are links that are correct, but generating errors for some reason, the [domains? links? The exclude list in this PR has domains] can be added to the exclude list in \`book.toml\`."

Reply:
It's non optional in that you can not run it locally, then CI should catch any issues and you will have to solve them manually. What do you think?

Regarding the links, yes, I should specify domain. Web jargon is not my thing :p

Reply:
Well, i'm thinking about trying to reducing unnecessary work. We tell people to run `vmdfmt` on their contribution before submitting so that everyone involved doesn't have to go through the dance of the submitter being told that the CI check has failed and that they need to fix the PR; i'd like to take the same approach with `mdbook-linkcheck`.

Reply:
Fair enough. It's in void-packages already so it shouldn't be too much of an issue. I will reword it :)